plugin/trino-clickhouse/pom.xml

```diff
@@ -113,6 +112,14 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <!-- Clickhouse client will by default use LZ4 compression, which requires this dependency -->
+            <!-- https://github.com/ClickHouse/clickhouse-docs/pull/2408/files -->
+            <groupId>org.lz4</groupId>
+            <artifactId>lz4-java</artifactId>
+            <scope>runtime</scope>
```

Comment:
Won't it break the compatibility with older server ? If yes we should update them in the docs right ?

Reply:
It should not as this PR bumps and tests via the oldest supported Clickhouse LTS version of the server `24.3.14.35`
If for some reason someone uses even older version, they may try to pass `compress=0` within connection parameters.

Reply:
captured here: https://github.com/trinodb/trino/pull/24515/files#diff-9887d9881d35f8487cd38095151074100f321413a2215fb1d92b352296bf55b1R22

Reply:
Also it's not an issue even for previously oldest supported `23.8.12.13`. or altinity `22.3`

Reply:
Actually I meant should we capture `compress=0` for older version for Clickhouse server. If older version also supports LZ4 compression they we could skip it.

Reply:
This code works for 
> 23.8.12.13. or altinity 22.3

versions without `compress=0`